tasks/section2.yml

```diff
@@ -1,146 +1,278 @@
-- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled"
-  command: /bin/true
-  register: result
+- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled | chargen-dgram"
+  shell: "chkconfig --list | grep chargen-dgram |grep -q on"
+  register: chargen1_server_enabled_audit
+  failed_when: chargen1_server_enabled_audit.rc == 0
   always_run: yes
   changed_when: no
   ignore_errors: yes
   tags:
       - level1
-      - level2
       - audit
       - rule_2.1.1
 
-- name: "SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled"
-  command: /bin/true
+- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled | chargen-stream"
+  shell: "chkconfig --list | grep chargen-stream |grep -q on"
+  register: chargen2_server_enabled_audit
+  failed_when: chargen2_server_enabled_audit.rc == 0
+  always_run: yes
+  changed_when: no
+  ignore_errors: yes
   tags:
       - level1
-      - level2
+      - audit
+      - rule_2.1.1
+
+- name: "SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled | chargen-dgram"
+  command: "chkconfig chargen-dgram off"
```

Comment:
Should use the `service` module here if possible. Same comment for 2.1.1 thru 2.1.6 patch tasks.